config/deploy.rb

```diff
@@ -153,4 +153,13 @@
       end
     end
   end
+  
+  desc 'Update manager fees to Stake Pools'
```

Comment:
You need to execute rake task within app folder - it's not systemctl service, it's rails task. So the correct syntax would be:
````
  desc 'Update manager fees to Stake Pools'
  task :update_fee_in_stake_pools do
    on release_roles([:all]) do
      within release_path do
        with rails_env: fetch(:rails_env) do
          execute :rake, 'update_fee_in_stake_pools:mainnet'
        end
      end
    end
  end
```